apps/site/assets/ts/schedule/components/schedule-finder/daily-schedule/DailySchedule.tsx

```diff
@@ -72,39 +72,41 @@ const SchedulesSelect = ({
 
   return (
     <div className="schedule-finder__service-selector">
-      <label htmlFor="service_selector" className="sr-only">
-        Schedules
+      <label>
+        <span className="sr-only">
+          Choose a schedule type from the available options
+        </span>
+        <SelectContainer>
+          <select
+            className="c-select-custom text-center u-bold"
+            defaultValue={defaultSelectedServiceId}
+            onChange={e =>
+              onSelectService(sortedServices.find(s => s.id === e.target.value))
+            }
+            aria-controls="daily-schedule"
```

Comment:
This attribute explicitly indicates that this `<select>` controls the daily schedules section.